index.js

```diff
@@ -506,10 +507,19 @@ class GithubScm extends Scm {
         // For pull requests
         if (config.prRef) {
             const prRef = config.prRef.replace('merge', 'head:pr');
+            const prNumber = prRef.split('/')[1];
+
+            // eslint-disable-next-line no-useless-escape
+            const regex = '\'s\/^\\[{\\(.*\\)}]$\/\\1\/\'';
 
             // Fetch a pull request
             command.push(`echo Fetching PR and merging with ${branch}`);
             command.push(`$SD_GIT_WRAPPER "git fetch origin ${prRef}"`);
+
+            // eslint-disable-next-line no-useless-escape
+            // eslint-disable-next-line max-len
+            command.push(`export PR_BRANCH_NAME=origin/$(curl -s https://api.${config.host}/repos/${config.org}/${config.repo}/pulls/${prNumber} | sed ${regex} | tr ',' '\n' | grep -m1 'ref' | cut -f 2 -d ':' | tr -d '\"' | tr -d ' ')`);
```

Comment:
I feel this is poor in reliability. It is better to use jq. Is there any reason you don't want to use jq?

Reply:
I first tried using `jq` command.
However, the `jq` command may not be usable because it does not exist in the image.
Therefore, I added a process to install the `jq` command, but I found that the operation became slow.

Reply:
Since there are refs in both the `base` and the `head`, I think it is necessary to clarify at least which ref is being referenced.
https://developer.github.com/v3/pulls/#get-a-single-pull-request

If there is a possibility of hitting `href` etc, you will also need to grep for an exact match.

Reply:
How about gets PR head branch name in the [getPrInfo](https://github.com/screwdriver-cd/scm-github/blob/master/index.js#L1298-L1338) and returns it.
In this method, gets pull request information by requesting github API.
And API returns exact single pr information [like this](https://developer.github.com/v3/pulls/#get-a-single-pull-request).
In this response body, `head.ref` is the PR head branch name.

Reply:
@kumada626 
Thank you review.  
I'll try that way.